Export type information

**Is your feature request related to a problem?**
Kinda. I'm trying to use the `opentelemetry-sdk` in my `mypy` enabled project and get the following error message:

```txt
uogc/trace.py:1: error: Skipping analyzing 'opentelemetry.sdk.trace': found module but no type hints or library stubs
```

**Describe the solution you'd like**
I see that several packages in this repository is using types. However, these types are not published. Publishing the types is actually not that tricky. We just did it for another project, where the dialogue and code changes are referred to in this PR:

https://github.com/developmentseed/geojson-pydantic/issues/5

In addition, the PyPi `"Typing :: Typed"` tag should probably be included. See https://github.com/hawkaa/pygeojson/blob/master/pyproject.toml#L14 .

**Describe alternatives you've considered**
I did think about distributing a separate stub library, but all the types are actually available in the original code. Publishing the type information should be an easy change.

**Additional context**
I don't know how many packages in this repository that actually has type information, and whether it is going to much work to change the workflow in this repository to add the `py.typed` file and do some changes in the package info.

Let me know what you think and if you have any questions!

--- a/exporter/opentelemetry-exporter-jaeger-proto-grpc/setup.cfg
+++ b/exporter/opentelemetry-exporter-jaeger-proto-grpc/setup.cfg
@@ -32,6 +32,7 @@ classifiers =
     Programming Language :: Python :: 3.7
     Programming Language :: Python :: 3.8
     Programming Language :: Python :: 3.9
+    Typing :: Typed
 
 [options]
 python_requires = >=3.6

--- a/exporter/opentelemetry-exporter-jaeger-thrift/setup.cfg
+++ b/exporter/opentelemetry-exporter-jaeger-thrift/setup.cfg
@@ -32,6 +32,7 @@ classifiers =
     Programming Language :: Python :: 3.7
     Programming Language :: Python :: 3.8
     Programming Language :: Python :: 3.9
+    Typing :: Typed
 
 [options]
 python_requires = >=3.6

--- a/exporter/opentelemetry-exporter-jaeger/setup.cfg
+++ b/exporter/opentelemetry-exporter-jaeger/setup.cfg
@@ -32,6 +32,7 @@ classifiers =
     Programming Language :: Python :: 3.7
     Programming Language :: Python :: 3.8
     Programming Language :: Python :: 3.9
+    Typing :: Typed
 
 [options]
 python_requires = >=3.6

--- a/exporter/opentelemetry-exporter-opencensus/setup.cfg
+++ b/exporter/opentelemetry-exporter-opencensus/setup.cfg
@@ -32,6 +32,7 @@ classifiers =
     Programming Language :: Python :: 3.7
     Programming Language :: Python :: 3.8
     Programming Language :: Python :: 3.9
+    Typing :: Typed
 
 [options]
 python_requires = >=3.6

--- a/exporter/opentelemetry-exporter-otlp/setup.cfg
+++ b/exporter/opentelemetry-exporter-otlp/setup.cfg
@@ -32,6 +32,7 @@ classifiers =
     Programming Language :: Python :: 3.7
     Programming Language :: Python :: 3.8
     Programming Language :: Python :: 3.9
+    Typing :: Typed
 
 [options]
 python_requires = >=3.6

--- a/exporter/opentelemetry-exporter-zipkin-json/setup.cfg
+++ b/exporter/opentelemetry-exporter-zipkin-json/setup.cfg
@@ -33,6 +33,7 @@ classifiers =
     Programming Language :: Python :: 3.7
     Programming Language :: Python :: 3.8
     Programming Language :: Python :: 3.9
+    Typing :: Typed
 
 [options]
 python_requires = >=3.5

--- a/exporter/opentelemetry-exporter-zipkin-proto-http/setup.cfg
+++ b/exporter/opentelemetry-exporter-zipkin-proto-http/setup.cfg
@@ -33,6 +33,7 @@ classifiers =
     Programming Language :: Python :: 3.7
     Programming Language :: Python :: 3.8
     Programming Language :: Python :: 3.9
+    Typing :: Typed
 
 [options]
 python_requires = >=3.5

--- a/exporter/opentelemetry-exporter-zipkin/setup.cfg
+++ b/exporter/opentelemetry-exporter-zipkin/setup.cfg
@@ -32,6 +32,7 @@ classifiers =
     Programming Language :: Python :: 3.7
     Programming Language :: Python :: 3.8
     Programming Language :: Python :: 3.9
+    Typing :: Typed
 
 [options]
 python_requires = >=3.6

--- a/opentelemetry-api/setup.cfg
+++ b/opentelemetry-api/setup.cfg
@@ -32,6 +32,7 @@ classifiers =
     Programming Language :: Python :: 3.7
     Programming Language :: Python :: 3.8
     Programming Language :: Python :: 3.9
+    Typing :: Typed
 
 [options]
 python_requires = >=3.6

--- a/opentelemetry-distro/setup.cfg
+++ b/opentelemetry-distro/setup.cfg
@@ -31,6 +31,7 @@ classifiers =
     Programming Language :: Python :: 3.6
     Programming Language :: Python :: 3.7
     Programming Language :: Python :: 3.8
+    Typing :: Typed
 
 [options]
 python_requires = >=3.6

--- a/opentelemetry-sdk/setup.cfg
+++ b/opentelemetry-sdk/setup.cfg
@@ -32,6 +32,7 @@ classifiers =
     Programming Language :: Python :: 3.7
     Programming Language :: Python :: 3.8
     Programming Language :: Python :: 3.9
+    Typing :: Typed
 
 [options]
 python_requires = >=3.6

--- a/propagator/opentelemetry-propagator-b3/setup.cfg
+++ b/propagator/opentelemetry-propagator-b3/setup.cfg
@@ -32,6 +32,7 @@ classifiers =
     Programming Language :: Python :: 3.7
     Programming Language :: Python :: 3.8
     Programming Language :: Python :: 3.9
+    Typing :: Typed
 
 [options]
 python_requires = >=3.6

--- a/propagator/opentelemetry-propagator-jaeger/setup.cfg
+++ b/propagator/opentelemetry-propagator-jaeger/setup.cfg
@@ -32,6 +32,7 @@ classifiers =
     Programming Language :: Python :: 3.7
     Programming Language :: Python :: 3.8
     Programming Language :: Python :: 3.9
+    Typing :: Typed
 
 [options]
 python_requires = >=3.6

--- a/shim/opentelemetry-opentracing-shim/setup.cfg
+++ b/shim/opentelemetry-opentracing-shim/setup.cfg
@@ -32,6 +32,7 @@ classifiers =
     Programming Language :: Python :: 3.7
     Programming Language :: Python :: 3.8
     Programming Language :: Python :: 3.9
+    Typing :: Typed
 
 [options]
 python_requires = >=3.6

